pandas/tseries/offsets.py

```diff
@@ -2219,6 +2219,9 @@ class Tick(SingleConstructorOffset):
     def __init__(self, n=1, normalize=False):
         # TODO: do Tick classes with normalize=True make sense?
```

Comment:
It seems like this was a broader API design question.  Depending on how things go, creating an issue to "close" with this PR might be appropriate.

Reply:
@jbrockmendel : Could you remove this comment?  Looks like your fix is here to say. 😄